### PR TITLE
Removes native auction/live auction conflation

### PR DESCRIPTION
**Pull request by ashfurrow**
We were using a check for native auctions to gate live auctions, which we didn't need since that's handled by echo routing.

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Warning</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Podfile [hunk 1]
```diff
@@ -125,8 +125,8 @@ target 'Artsy Tests' do
   pod 'Forgeries/Mocks', :git => "https://github.com/ashfurrow/Forgeries.git", :branch => "application"
 
   # Swift pods 🎉
-  pod 'Quick'
-  pod 'Nimble'
+  pod 'Quick', git: 'https://github.com/Quick/Quick.git'
```
Comment (ashfurrow) [inline Podfile]: Removes some warnings from new Xcode 7.3 (they haven't cut a release yet).

File: Artsy/App/ARSwitchBoard.m [hunk 2]
```diff
@@ -206,13 +206,11 @@ - (void)updateRoutes
 
     Route *route = self.echo.routes[@"ARLiveFairsURLDomain"];
```
Comment (ashfurrow) [inline Artsy/App/ARSwitchBoard.m]: As I understand, the feature flag is unnecessary, sine we can just remove the `ARLiveFairsURLDomain` and it should fall through to a martsy view.

Comment (alloy) [inline Artsy/App/ARSwitchBoard.m]: Yep, seems so.

File: Artsy/App/ARAppStatus.m [hunk 3]
```diff
@@ -45,7 +45,7 @@ + (BOOL)isRunningTests
     static BOOL isRunningTests = NO;
     static dispatch_once_t onceToken = 0;
     dispatch_once(&onceToken, ^{
-        isRunningTests = [[NSProcessInfo processInfo] environment][@"XCInjectBundle"] != nil;
+        isRunningTests = NSClassFromString(@"XCTestCase") != nil;
```
Comment (ashfurrow) [inline Artsy/App/ARAppStatus.m]: I suppose technically this should be comparing `!= NULL`

File: Artsy/Categories/Apple/UIDevice-Hardware.m [hunk 4]
```diff
@@ -378,9 +378,7 @@ + (BOOL)isRunningUnitTests
 {
     static dispatch_once_t oncePredicate;
     dispatch_once(&oncePredicate, ^{
-        NSString *XCInjectBundle = [[[NSProcessInfo processInfo] environment] objectForKey:@"XCInjectBundle"];
-
-        ARRunningUnitTests = [XCInjectBundle hasSuffix:@".xctest"];
+        ARRunningUnitTests = NSClassFromString(@"XCTestCase") != nil;
```
Comment (alloy) [inline Artsy/Categories/Apple/UIDevice-Hardware.m]: Jeez, how many places do we need to define this, ey? Do you know if this stuff is actually used? I think I checked at some point and pretty much nothing from this file was used except for the `isPad` stuff.

Comment (ashfurrow) [inline Artsy/Categories/Apple/UIDevice-Hardware.m]: Not sure, I just did a project-wide search for `XCInjectBundle`, which I probably should've done yesterday :grimacing:

Comment (ashfurrow) [inline Artsy/Categories/Apple/UIDevice-Hardware.m]: Doesn't appear to be called from anywhere.

Comment (alloy) [inline Artsy/Categories/Apple/UIDevice-Hardware.m]: Can you rm all of the file except the `isPad` method?

Comment (ashfurrow) [inline Artsy/Categories/Apple/UIDevice-Hardware.m]: :+1:

**Comment by ashfurrow** [comment]
Cool, switching back and forth from this PR and #1326 has given me some ideas on why that one was failing. Once this PR is merged and I can pull changes on that one, I'll verify.

File: Artsy/App/ARAppStatus.m [hunk 5]
```diff
@@ -45,7 +45,7 @@ + (BOOL)isRunningTests
     static BOOL isRunningTests = NO;
     static dispatch_once_t onceToken = 0;
     dispatch_once(&onceToken, ^{
-        isRunningTests = NSClassFromString(@"XCTestCase") != nil;
+        isRunningTests = NSClassFromString(@"XCTestCase") != NULL;
```
Comment (alloy) [inline Artsy/App/ARAppStatus.m]: I think you’re actually supposed to compare against `Nil`, but I definitely have never done so :stuck_out_tongue_closed_eyes:

**Comment by ashfurrow** [comment]
Only a changelog failure, up to you @alloy if you want to merge or hold off.

**Comment by alloy** [comment]
Weird that Danger does not have write access to the PR.

**Comment by alloy** [comment]
I’m fine with this not having a CHANGELOG :+1:

**Pull request merged**